src/ontology/mondo-edit.obo

```diff
@@ -353252,15 +353225,13 @@ consider: MONDO:0008394
 
 [Term]
 id: MONDO:0018337
-name: severe neonatal lactic acidosis due to NFS1-ISD11 complex deficiency
-comment: Reason of obsoletion: out of scope - MONDO:excludeGroupingClass. This is part of a large mass-obsoletion effort for ORDO terms.
-subset: obsoletion_candidate
+name: obsolete severe neonatal lactic acidosis due to NFS1-ISD11 complex deficiency
 subset: ordo_disease {source="Orphanet:397593"}
 xref: ICD10CM:E88.8 {source="Orphanet:397593", source="Orphanet:397593/attributed", source="Orphanet:397593/ntbt"}
```

Comment:
should this `xref` be maintained?

Reply:
oh, so it's ok to have some xref assertions maintained on obsoleted terms?

Reply:
based on another comment in this PR, it is ok to maintain xrefs for obsolete classes.